packages/client/tests/functional/dotenv-loading/_matrix.ts

```diff
@@ -0,0 +1,4 @@
+import { defineMatrix } from '../_utils/defineMatrix'
```

Comment:
Does not seem to be related to the rest of this PR?

Reply:
Oopsie I need to remove that